TinyConstraints/Classes/TinyEdgeInsets.swift

```diff
@@ -28,7 +28,7 @@
     import UIKit
 #endif
 
-extension TinyEdgeInsets {
+@MainActor extension TinyEdgeInsets {
```

Comment:
It's probably not recommended to add this since `TinyEdgeInsets` itself isn't isolated to the `@MainActor` and there's no code in any of these members that _requires_ `@MainActor` isolation